app/build.gradle

```diff
@@ -21,13 +21,13 @@ plugins {
 }
 
 android {
-    compileSdkVersion 30
-    buildToolsVersion "30.0.2"
+    compileSdkVersion 31
+    buildToolsVersion "30.0.3"
```

Comment:
You can actually delete this buildToolsVersion line now because I think it still works without it